charts/keycloakx/templates/statefulset.yaml

```diff
@@ -126,6 +126,10 @@ spec:
             - name: KC_METRICS_ENABLED
               value: "true"
             {{- end }}
+            {{- if .Values.metrics.enabled }}
+            - name: KC_HEALTH_ENABLED
+              value: "true"
```

Comment:
Is there a reason to disable this check? I am thinking about using these health check endpoints for the readiness, startup and liveness probes. Or is the current approach for those probes still more appropriate?

Reply:
We currently use the `/auth` / `/auth/realms/master` as readiness and liveness probes.

This health endpoint just exposes the `/auth/health` endpoint that can be customized.
The keycloak health endpoint is disabled by default, see: https://www.keycloak.org/server/all-config#_health

Reply:
I was just wondering about the purpose of the health endpoint and thought it would be most useful for readiness and liveness probe use cases. (Especially as it seems to support `/health/ready` and `/health/live` explicitly, but I don't know what those endpoints really do.)

Reply:
I'd say, we stick to the "old" readiness and livenessProbes for now  and discuss switching to the /health endpoint in another thread.